app/build.gradle

```diff
@@ -214,6 +214,10 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation "androidx.room:room-runtime:${androidxRoomVersion}"
     implementation "androidx.room:room-rxjava3:${androidxRoomVersion}"
+
+    // Apple Silicon Mac workaround (https://issuetracker.google.com/issues/174695268#comment9)
+    kapt "org.xerial:sqlite-jdbc:3.34.0"
```

Comment:
Mmmh, what about not adding this? Would it be a problem for you to add this to your local workspace but to not commit it, so that you are able to build it on Silicon Mac but it does not impact the app's size and dependencies?

Reply:
Done in https://github.com/TeamNewPipe/NewPipe/pull/6955/commits/ad99b54a39611a6968f74d2e2276f92105fd2c20.

Sorry for adding this previously.